src/Tribe/Views/V2/Widgets/Widget_QR_Code.php

```diff
@@ -235,10 +233,18 @@ public function setup_admin_fields() {
 			];
 
 			$series = get_posts( $args );
-			foreach ( $series as $series ) {
+			if ( ! empty( $series ) ) {
+				$series_options = [];
+				foreach ( $series as $series ) {
+					$series_options[] = [
+						'value' => $series->ID,
+						'text'  => $series->ID . ' - ' . $series->post_title,
+					];
+				}
+			} else {
 				$series_options[] = [
-					'value' => $series->ID,
-					'text'  => $series->ID . ' - ' . $series->post_title,
+					'value' => '',
+					'text'  => _x( 'There are no Series created yet.', 'No series created yet', 'the-events-calendar' ),
```

Comment:
I don't know that context is needed here, as the context value doesn't provide any new information to someone translating the string.

Also, a minor suggestion for the text itself.

```suggestion
					'text'  => __( 'No Series have been created yet.', 'the-events-calendar' ),
```